investigations/Possible_Instrument_View_Improvements.md

```diff
@@ -0,0 +1,67 @@
+# Investigation of Possible Instrument 2.0 Effects on Instrument View Performance.
+This document outlines areas in which the performance of the instrument view may benefit from the introduction of the new Instrument 2.0 caching layers. Tests were carried out using the Microsoft Visual Studio debugger and profiling tools in conjunction with `Callgrind` on Linux which provided information on which Instrument 1.0 methods are most heavily used when interacting with the instrument view. The instrumentation was taken for the duration of the creation of the `InstrumentWidget`.
+
+## Findings
+
+The following report contains results in terms of percentage inclusive time during the loading of an empty GEM instrument in the instrument view:
```

Comment:
Yes, on visual studio, even with instrumentation switched on for `Geometry` and `MantidQtWidgetsInstrumentView` the majority of time was spent as detailed in this document. Instrument 1.0 geometry access seems to dominate the processing time.